private/bufpkg/bufconfig/buf_gen_yaml_file.go

```diff
@@ -496,11 +496,12 @@ type externalGeneratePluginConfigV2 struct {
 	// Local is the local path (either relative or absolute) to a binary or other runnable program which
 	// implements the protoc plugin interface. This can be one string (the program) or multiple (remaining
 	// strings are arguments to the program).
-	Local interface{} `json:"local,omitempty" yaml:"local,omitempty"`
+	Local any `json:"local,omitempty" yaml:"local,omitempty"`
```

Comment:
Should we change `Opt` to `any` as well while we're here?

Reply:
yeah, good catch